wiki/user/information-center.md

```diff
@@ -9,46 +9,43 @@ parent: wiki
 
 Downloaded ACE3 and have no idea where to start? This page serves as a starting point to help new players and mission makers understand what's available to them.
 
+**Q:** Don't know where to begin your ACE3 journey?</br>
+**A:** [**Check out ACE3 features**](http://ace3mod.com/wiki/feature/)
 
-- You don't know where to begin your ACE3 journey? [**Check out ACE3 features**](http://ace3mod.com/wiki/feature/)
-
-
-- You are a mission maker but you don't know what ACE3 has to offer? [**We have some documentation for you**](http://ace3mod.com/wiki/missionmaker/)
-
-
-- Are you searching for ACE3 classnames ? [**Here they are**](http://ace3mod.com/wiki/missionmaker/classnames.html)
+**Q:** Are you a mission maker but don't know yet what ACE3 has to offer?</br>
+**A:** [**We have a wiki section just for you**](http://ace3mod.com/wiki/missionmaker/)
 
+**Q:** Looking for information on ACE3 class names?</br>
+**A:** [**Here they are**](http://ace3mod.com/wiki/missionmaker/class-names.html)
 
 ## 1. FAQ
 ### 1.1 Features
-**Q:** Where is X feature? </br>
-**A:** When it's done.</br>
+**Q:** Where is X feature?</br>
```

Comment:
Use markdown instead of html tags. In this case double space at the end of line.